book/src/backends/README.md

```diff
@@ -0,0 +1,3 @@
+# Backends
+
+powdr aims to have full flexibility when it comes to generating proofs. However, powdr comes built-in with a few backends to get started with zkVMs.
```

Comment:
I think "however" sounds a bit negative here. How about just

```suggestion
powdr aims to have full flexibility when it comes to generating proofs and comes with a few built-in backends to get started with zkVMs.
```

Reply:
had the same thought!